Adds OVNK secondary network release note (#59330)
openshift/openshift-docs: release_notes/ocp-4-13-release-notes.adoc

* Installer-provisioned infrastructure: xref:../installing/installing_bare_metal_ipi/ipi-install-installation-workflow.adoc#modifying-install-config-for-dual-stack-network_ipi-install-installation-workflow[Deploying with dual-stack networking]

> * User-provisioned infastructure: xref:../installing/installing_bare_metal/installing-bare-metal-network-customizations.adoc#installation-configuration-parameters-network_installing-bare-metal-network-customizations[Network configuration parameters]

Reviewer: ```suggestion
* User-provisioned infastructure: xref:../installing/installing_bare_metal/installing-bare-metal-network-customizations.adoc#installation-configuration-parameters-network_installing-bare-metal-network-customizations[Network configuration parameters].
```

[id="ocp-4-13-ovn-kubernetes-plugin-secondary-network"]
==== OVN-Kubernetes is available as a secondary network (Technology Preview)

> With this release, the {openshift-networking} OVN-Kubernetes network plugin allows the configuration of secondary network interfaces for pods. As a secondary network, OVN-Kubernetes supports a layer 2 (switched) topology network. This is available as a Technology Preview feature.

Reviewer: ```suggestion
With this release, the {openshift-networking} OVN-Kubernetes network plug-in allows the configuration of secondary network interfaces for pods. As a secondary network, OVN-Kubernetes supports a layer 2 (switched) topology network. This is available as a Technology Preview feature.
```
See the ISG: https://www.ibm.com/docs/en/ibm-style?topic=word-usage#p  (There have been a few discussions about 'plugin' versus 'plug-in'...I assume the latest directive is to use 'plug-in'.

Author: I think the latest directive is to use "plugin" lol: https://redhat-documentation.github.io/supplementary-style-guide/#plugin

> For more information on OVN-Kubernetes as a secondary network, see xref:../networking/multiple_networks/configuring-additional-network.html#configuration-ovnk-additional-networks_configuring-additional-network[Configuration for an OVN-Kubernetes additional network]

Reviewer: ```suggestion
For more information about OVN-Kubernetes as a secondary network, see xref:../networking/multiple_networks/configuring-additional-network.html#configuration-ovnk-additional-networks_configuring-additional-network[Configuration for an OVN-Kubernetes additional network]
```
See ISG: https://www.ibm.com/docs/en/ibm-style?topic=word-usage#i

|Not Available
|Technology Preview

> |OVN-Kubernetes network plugin as secondary network

Reviewer: ```suggestion
|OVN-Kubernetes network plug-in as secondary network
```
See the ISG: https://www.ibm.com/docs/en/ibm-style?topic=word-usage#p
